### DISCUSSION TITLE
About the github Education Student Pack application

### DISCUSSION THREAD

**Discussion by haloplove** [community/community#52249] [GitHub Education]
I have tried my school acceptance letter, student ID, online proof of enrollment, class schedule, and grades, but the system still does not recognize it and says "Not enough to prove your current academic status." My application has been rejected on the grounds that it is "insufficient to prove your current academic status" and I would like to have my application manually reviewed.
![image](https://user-images.githubusercontent.com/84855171/230601002-877a13c9-efbf-4aac-8845-4aa47403238a.png)

**Comment by samus-aran** [comment]
Hi @haloplove 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍

**Reply by haloplove** [reply]
Thank you very much for your reply, at present I have applied through, although I do not know what factors led to the success, I still hope that the automatic review can be a little more intelligent, so as not to disturb you because of these things .

**Reply by samus-aran** [reply]
Thank you for being considerate of my time 😅 Seeing what problems our students have when applying we're able to forward that to the teams to improve those areas as well as documentation.

I'm so glad you were successful 🎉 What offer or area of GitHub Global Campus are you most excited to try first? 🥇

**Reply by haloplove** [reply]
Thank you for checking out my proposal. As a student, I would like to thank your team for giving me the opportunity to be exposed to the world's advanced technology. I am currently learning about IoT and hope to experience the wonders of IoT + ai. So after applying for the student package, I also applied for the copilot x project. After I finish my IoT car project, I also want to give back to the github community.

**Reply by samus-aran** [reply]
Love it! ❤️ Please do stick around the GitHub community, I'm sure your learning in IoT and AI would be very interesting to others. We also just launched a new [Machine Learning](https://github.com/orgs/community/discussions/categories/machine-learning-and-mlops) and [Programming Help](https://github.com/orgs/community/discussions/categories/programming-help) category if there are some insightful conversations there.
